packages/user-operation-controller/src/utils/validation.ts

```diff
@@ -140,6 +140,9 @@ export function validateUpdateUserOperationResponse(
   const ValidResponse = optional(
     object({
       paymasterAndData: optional(HexOrEmptyBytes),
+      callGasLimit: optional(HexStruct),
```

Comment:
We actually already have a local type for this provided by `defineHex`.

Can we use that for now and then we'll make a separate future PR to use the types provided by `@metamask/utils`?